.github/workflows/publish-docker.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish Docker Image
```

Comment:
This workflow is only used in one repo, so I think we could forego adding it to this repository

Reply:
What do you think about private notes transport? Probably, it would be convenient to have the docker image as well for other repos with binaries artefacts as well?